docs/cimpl6LanguageReference.md

```diff
@@ -418,11 +405,11 @@ Value:             concept from YesNoUnknownVS (required)
 ````
 For more information on binding, see [Value Set Binding Constraint](#value-set-binding-constraint).
 
->**Note:** CIMPL 5.0 Grammar associating a cardinality to a `Value` is obsolete.
+>**Note:** You cannot assign a cardinality to a `Value`. Cardinalities can only be introduced in `Property` declarations.
 
 ### ValueSet
 
-Defines the name of a value set defined inside CIMPL (references to external value sets are also allowed). By convention, all value set names should be Pascal case, starting with the name of the corresponding concept data type Element that is being bound to the value set, and ending with _VS_.
+Defines the name of a value set inside CIMPL value set files. For details on the usage of this statement, see [Value Set File](#value-set-file). 
```

Comment:
For details on the use of this statement      (change usage to use)

Reply:
fixed.